src/hotspot/share/opto/escape.cpp

```diff
@@ -1128,7 +1145,7 @@ void ConnectionGraph::process_call_arguments(CallNode *call) {
               es = PointsToNode::NoEscape;
             }
           }
-          set_escape_state(arg_ptn, es);
+          set_escape_state(arg_ptn, es NOT_PRODUCT(COMMA "reason unknown (1)"));
```

Comment:
Transition here is `NoEscape` -> `ArgEscape`. So the reason could be "is argument".

Reply:
Thanks for the suggestion. I think we can use `trace_arg_escape_message` here to also print out the relevant call node.